### PR TITLE
bsp: Added LCD support into ESP32-S3-Korvo-2 Board Support Package

### PR DESCRIPTION
**Pull request by espzav**
Added LCD support into Korvo-2 BSP. 
I decided change the audio settings back to mono. 

Tested with:
- display_rotation
- display_camera
- display_audio_photo

### CODE REVIEW

**Comment by espzav** [comment]
@tore-espressif PTAL

File: esp32_s3_korvo_2/esp32_s3_korvo_2.c [hunk 1]
```diff
@@ -577,3 +418,27 @@ esp_err_t bsp_led_set(const bsp_led_t led_io, const bool on)
     BSP_ERROR_CHECK_RETURN_ERR(esp_io_expander_set_level(io_expander, led_io, !on));
     return ESP_OK;
 }
+
+esp_err_t bsp_button_init(const bsp_button_t btn)
+{
+    //TODO: initialize IOT buttons and handle it in BSP
+    return ESP_OK;
```
Comment (tore-espressif) [inline esp32_s3_korvo_2/esp32_s3_korvo_2.c:425]: Do you have any plans with this TODO?

**Review by tore-espressif — COMMENTED** [review]
One comment about the buttons support. Otherwise LGTM!

File: esp32_s3_korvo_2/idf_component.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ targets:
 
 dependencies:
   idf: ">=5.0"
+  esp_lcd_ili9341: "^1"
   esp_lcd_touch_tt21100: "^1"
   esp_io_expander_tca9554: "^1"
```
Comment (tore-espressif) [inline esp32_s3_korvo_2/idf_component.yml:12]: I'm having issues with `esp_io_expander`. Should it be listed here as 'public' dependency? Its header file is in in public `esp32_s3_korovo_2.h` header file.

See related job: https://github.com/espressif/esp-bsp/actions/runs/4067908226/jobs/7005819842

Comment (tore-espressif) [reply esp32_s3_korvo_2/idf_component.yml:12]: cc @Lzw655

File: esp32_s3_korvo_2/esp32_s3_korvo_2.c (hunk 1 above)
Comment (espzav) [reply esp32_s3_korvo_2/esp32_s3_korvo_2.c]: If you remember, we told about it before two weeks. I removed TODO and I will put it into JIRA.

**Comment by espzav** [comment]
@tore-espressif Thank you for the review!

File: esp32_s3_korvo_2/esp32_s3_korvo_2.c (hunk 1 above)
Comment (tore-espressif) [reply esp32_s3_korvo_2/esp32_s3_korvo_2.c]: Great, thank you!

File: esp32_s3_korvo_2/idf_component.yml (hunk 2 above)
Comment (tore-espressif) [reply esp32_s3_korvo_2/idf_component.yml]: Works OK now :)

**Review by tore-espressif — APPROVED** [review]
Please rebase on master and clean your commit history. Then LGTM!

**Pull request merged**